gateway/src/apicast/template_string.lua

```diff
@@ -63,8 +63,7 @@ for name, func in pairs(liquid_filters) do
   liquid_filter_set:add_filter(name, func)
 end
 
--- Set resource limits to avoid loops
-local liquid_resource_limit = LiquidResourceLimit:new(nil, nil, 0)
+local liquid_resource_limit
```

Comment:
In case Liquid is actually needed, there is no need for this variable to be defined on the top level.

Reply:
Removed changes.